uaclient/util.py

```diff
@@ -676,10 +676,12 @@ def subp(
         except ProcessExecutionError as e:
             if capture:
                 logging.debug(str(e))
+                print(str(e))
             if not retry_sleeps:
                 raise
             retry_msg = " Retrying %d more times." % len(retry_sleeps)
             logging.debug(redact_sensitive_logs(str(e) + retry_msg))
+            print(str(e))
```

Comment:
Should we only print that message if `capture` is `True` ? Otherwise we could be printing this message twice to the user